kolibri/core/assets/src/views/SidePanelModal/index.vue

```diff
@@ -252,7 +252,7 @@
   @import '~kolibri-design-system/lib/styles/definitions';
 
   .header-content {
-    width: 100%;
+    width: 90%;
```

Comment:
Looking around, it seems we've started accepting usage of CSS `calc()` in Kolibri. That could be an option here, since the 10% you removed would vary depending on width. So something like:
```css
width: calc(100% - 30px);
```

Reply:
Just using `30px` as an example, I'm not sure how much space is actually needed

Reply:
Yes, that's a good point. All of the titles will not be the same width. I will test a few different numbers with the different titles to find an appropriate amount of space to subtract!

Reply:
These seems like an unproblematic use of `calc` - there are some caveats for `calc` in IE11 (look under the known issues tab): https://caniuse.com/?search=calc, but they don't seem to apply here!